challenge_2/python/ekstrah/src/a.py

```diff
@@ -1,18 +1,17 @@
-def findingMajoList(aList): #{
-    bList = set(aList)
-    bList = list(bList)
-    for i in range(0, len(bList)): #{
-        if aList.count(bList[i]) > len(bList): #{
-            return bList[i]
+def findingSingleList(aList): #{
+    bList = list(set(aList))
+    for item in bList: #{
+        if aList.count(item) == 1: #{
```

Comment:
The `.count()` method of a list iterates over the entire list which causes your solution complexity to be O(n^2) see if you can think of a way to reduce the complexity to O(n).